### ISSUE TITLE
Infinite Recent Changes

### ISSUE THREAD

**Issue by alexdaniel654** [nottinghamtec/PyRIGS#547]
Large training sessions fill up recent changes, all the info that's there is useful, it would just be more useful if things didn't drop off the bottom (I tend to try and keep track of what people have been passed out on so I can run more appropriate training). Would it be possible to either make recent changes bigger or make it so it loads more changes as you scroll to the bottom?

I seem to remember someone saying this wouldn't actually be that difficult, if it is then just ignore me!

**Comment by FreneticScribbler** [comment]
The "activity streams" are infinite (paginated), it's just the 'recent changes' on the front page that gets truncated to things that are actually recent. Training sessions definetely hammer this, and I had been meaning to revise the UX but it's a bit of a pain. I could certainly make it longer, but I'd point you towards using https://rigs.nottinghamtec.co.uk/training/activity/ for this usecase. Does that work?

**Comment by FreneticScribbler** [comment]
Having looked at it, I need to revise the UX of the activity streams to be more useful for training, as "Created TrainingItemQualification" is delightfully unhelpful....

**Comment by alexdaniel654** [comment]
Yeah, having to click every item to see what's happened isn't really the "take a glance at the rigboard to see what people were trained in last night" level of information gathering I was hoping for...
